wgsl/index.bs

```diff
@@ -8127,17 +8127,29 @@ A WGSL program is a sequence of optional [=directives=] followed by [=module sco
 
 ## Limits ## {#limits}
 
-A program [=shader-creation error|must=] satisfy the following limits:
+A WGSL implementation will support shaders that satisfy the following limits.
+A WGSL implementation may support shaders that go beyond the specified limits.
+
+Note: A WGSL implementation should issue an error if it does not support a
+shader that goes beyond the specified limits.
 
 <table class='data'>
   <caption>Quantifiable shader complexity limits</caption>
   <thead>
-    <tr><th>Limit<th>Maximum value
+    <tr><th>Limit<th>Minimum supported value
   </thead>
-    <tr><td>Number of members in a [=structure=] type<td>16383
-    <tr><td>[=Nesting depth=] of a [=composite=] type<td>255
-    <tr><td>Number of [=formal parameter|parameters=] for a function<td>255
-    <tr><td>Number of case selector values in a [=statement/switch=] statement<td>16383
+    <tr><td>Maximum number of members in a [=structure=] type<td>16383
+    <tr><td>Maximum [=nesting depth=] of a [=composite=] type<td>255
+    <tr><td>Maximum number of [=formal parameter|parameters=] for a function<td>255
+    <tr><td>Maximum number of case selector values in a [=statement/switch=] statement<td>16383
+    <tr><td>Maximum [=byte-size=] of an [=array=] type instantiated in the following [=address spaces=]:
+            * [=address spaces/function=]
+            * [=address spaces/private=]
+            * [=address spaces/workgroup=]
```

Comment:
WebGPU has a 16KB limit on workgroup storage.
Suggest making a different row for workgroup storage, use 16KB and indicate that  https://gpuweb.github.io/gpuweb/#dom-supported-limits-maxcomputeworkgroupstoragesize is authoritative.